servers/rendering/renderer_viewport.cpp

```diff
@@ -1378,7 +1384,9 @@ bool RendererViewport::viewport_is_using_hdr_2d(RID p_viewport) const {
 void RendererViewport::viewport_set_screen_space_aa(RID p_viewport, RS::ViewportScreenSpaceAA p_mode) {
 	Viewport *viewport = viewport_owner.get_or_null(p_viewport);
 	ERR_FAIL_NULL(viewport);
-	ERR_FAIL_COND_EDMSG(p_mode != RS::VIEWPORT_SCREEN_SPACE_AA_DISABLED && OS::get_singleton()->get_current_rendering_method() == "gl_compatibility", "Screen space AA is currently unavailable on the Compatibility renderer.");
+	if (OS::get_singleton()->get_current_rendering_method() == "gl_compatibility" && p_mode != RS::VIEWPORT_SCREEN_SPACE_AA_DISABLED) {
+		WARN_PRINT_ONCE_ED("Screen-space AA is only available when using the Compatibility renderer.");
```

Comment:
This is backwards, it's *not* available on the compatibility renderer, so:
```suggestion
		WARN_PRINT_ONCE_ED("Screen-space AA is only available when using the Forward+ and Mobile renderers.");
```
Or keep the original message
```suggestion
		WARN_PRINT_ONCE_ED("Screen space AA is currently unavailable on the Compatibility renderer.");
```